dlp/api/Snippets/RedactDetectedTextInImage.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START dlp_redact_image_all_text]
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Dlp.V2;
+using Google.Protobuf;
+using System;
+using System.IO;
+
+public class RedactDetectedTextInImage
+{
+    public static RedactImageResponse RedactTextImage(
+        string projectId,
+        string originalImagePath,
+        string redactedImagePath)
+    {
+        // Instantiate the dlp client.
+        var dlp = DlpServiceClient.Create();
+
+        // Construct the content item.
```

Comment:
Please emulate the comments in Java sample

Reply:
Done